tests/tracer/test_forksafe.py

```diff
@@ -146,3 +150,44 @@ def f3():
     _, status = os.waitpid(pid, 0)
     exit_code = os.WEXITSTATUS(status)
     assert exit_code == 12
+
+
+if six.PY2:
+    lock_release_exc_type = _thread.error
+else:
+    lock_release_exc_type = RuntimeError
+
+
+def test_lock_basic():
```

Comment:
Can we add similar tests for `RLock` and `Event`?

Reply:
Will do!